From e6e05877f6e0c8ba363422cec33b8913497dbf78 Mon Sep 17 00:00:00 2001
From: Isaac Freund <ifreund@ifreund.xyz>
Date: Sat, 15 Aug 2020 14:57:41 +0200
Subject: [PATCH] meson: don't use .gz extension for man pages

The man pages aren't compressed with gzip and therefore shouldn't have
this extension.
---
 doc/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/meson.build b/doc/meson.build
index 7e3d9b4..2755dd2 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -4,7 +4,7 @@ scdoc_prog = find_program(scdoc.get_pkgconfig_variable('scdoc'), native: true)
 
 foreach man_src : ['fuzzel.scd']
   base = man_src.split('.')[-2]
-  out = '@0@.1.gz'.format(base)
+  out = '@0@.1'.format(base)
 
   custom_target(
     out,
-- 
2.28.0

